### PR TITLE
docs: Add alert setup for SAP Cloud Logging

### PR DESCRIPTION
**Pull request by a-thaler**
## Description

Changes proposed in this pull request (what was done and why):

- added recommended alerts for cloud logging
- added brief description on how to import alerts
- added brief description about available dashboards

Changes refer to particular issues, PRs or documents:

- 

## Traceability
- [ ] The PR is linked to a GitHub issue.
- [ ] New features have a milestone set.
- [ ] New features have defined acceptance criteria in a corresponding GitHub Issue, and all criteria are satisfied with this PR.
- [ ] The corresponding GitHub issue has a respective `area` and `kind` label.
- [ ] The follow-up issues (if any) are linked in the `Related Issues` section.
- [ ] Adjusted the documentation if the change is user-facing.
- [ ] The feature is unit-tested
- [ ] The feature is e2e-tested

### CODE REVIEW

File: docs/user/integration/sap-cloud-logging/README.md [hunk 1]
```diff
@@ -284,3 +285,31 @@ For easier access, add a navigation node to the Observability section as well as
...
+| Kyma Telemetry Integration | [Application log ingestion](https://raw.githubusercontent.com/kyma-project/telemetry-manager/main/docs/user/integration/sap-cloud-logging/alert-app-log-ingestion.json) | The LogPipeline for shipping [application logs](#ship-logs-to-sap-cloud-logging) might loose connectivity to SAP Cloud Logging resulting in no application logs being ingested anymore.|
+| Kyma Telemetry Integration | [Access log ingestion](https://raw.githubusercontent.com/kyma-project/telemetry-manager/main/docs/user/integration/sap-cloud-logging/alert-access-log-ingestion.json) | The LogPipeline for shipping [access logs](#ship-logs-to-sap-cloud-logging) might loose connectivity to SAP Cloud Logging resulting in no application logs being ingested anymore. |
+| Kyma Telemetry Integration | [Trace ingestion](https://raw.githubusercontent.com/kyma-project/telemetry-manager/main/docs/user/integration/sap-cloud-logging/alert-trace-ingestion.json) | The TracePipeline for shipping [traces](#ship-distributed-traces-to-sap-cloud-logging) might loose connectivity to SAP Cloud Logging resulting in no application logs being ingested anymore. |
```
Comment (chrkl) [inline docs/user/integration/sap-cloud-logging/README.md:308]:
```suggestion
| Kyma Telemetry Integration | [Trace ingestion](https://raw.githubusercontent.com/kyma-project/telemetry-manager/main/docs/user/integration/sap-cloud-logging/alert-trace-ingestion.json) | The TracePipeline for shipping [traces](#ship-distributed-traces-to-sap-cloud-logging) might loose connectivity to SAP Cloud Logging resulting in no distributed traces being ingested anymore. |
```

File: docs/user/integration/sap-cloud-logging/README.md [hunk 2]
```diff
@@ -284,3 +285,31 @@ For easier access, add a navigation node to the Observability section as well as
...
+| Kyma Telemetry Integration | [Metric ingestion](https://raw.githubusercontent.com/kyma-project/telemetry-manager/main/docs/user/integration/sap-cloud-logging/alert-metric-ingestion.json) | The MetricPipeline for shipping [metrics](#ship-metrics-to-sap-cloud-logging) might loose connectivity to SAP Cloud Logging resulting in no application logs being ingested anymore. |
```
Comment (chrkl) [inline docs/user/integration/sap-cloud-logging/README.md:309]:
```suggestion
| Kyma Telemetry Integration | [Metric ingestion](https://raw.githubusercontent.com/kyma-project/telemetry-manager/main/docs/user/integration/sap-cloud-logging/alert-metric-ingestion.json) | The MetricPipeline for shipping [metrics](#ship-metrics-to-sap-cloud-logging) might loose connectivity to SAP Cloud Logging resulting in no monitoring metrics being ingested anymore. |
```

File: docs/user/integration/sap-cloud-logging/README.md [hunk 3]
```diff
@@ -284,3 +285,31 @@ For easier access, add a navigation node to the Observability section as well as
...
+
+SAP Cloud Logging provides an alerting mechanism based on the OpenSearch Dashboards [alerting plugin](https://opensearch.org/docs/1.3/observing-your-data/alerting/index/). In the following it is described how to define and import recommended alerts.
+
+### Import
+
+The alerts defined in the subsequent chapters are based on JSON documents defining a `Monitor` for the alerting plugin. Such monitor work instantly after import, however the configured notification action is missing a `destination` which needs to be added manually afterwards. Also, adjust the intervals and thresholds to your needs as a general guidance cannot be given.
+
+To import a monitor, go to `Management > Dev Tools` in the Cloud Logging Dashboard and execute `POST _plugins/_alerting/monitors` followed by the contents of the respective JSON content. Afterwards, you will find the new monitor definition at `OpenSearch Plugins > Alerting`.
+
+### Recommended Alerts
```
Comment (chrkl) [inline docs/user/integration/sap-cloud-logging/README.md:299]: Maybe let's have some hint to enable only those alerts that have a corresponding pipeline deployed.

File: docs/user/integration/sap-cloud-logging/README.md [hunk 4]
```diff
@@ -284,3 +285,31 @@ For easier access, add a navigation node to the Observability section as well as
...
+
+In the following table you will find the definitions of recommended alerts with some basic description.
```
Comment (NHingerl) [inline docs/user/integration/sap-cloud-logging/README.md:301]:
```suggestion
The following alerts are recommended:
```

File: docs/user/integration/sap-cloud-logging/README.md [hunk 5]
```diff
@@ -284,3 +285,31 @@ For easier access, add a navigation node to the Observability section as well as
...
+
+| Category | File | Description |
+| -- | -- | -- |
+| Cloud Logging | [OpenSearch cluster health](https://raw.githubusercontent.com/kyma-project/telemetry-manager/main/docs/user/integration/sap-cloud-logging/alert-health.json) | The OpenSearch cluster itself can get in an unhealthy state which gets indicated by a "red" status using the [cluster health](https://opensearch.org/docs/1.3/api-reference/cluster-api/cluster-health) API.|
```
Comment (NHingerl) [inline docs/user/integration/sap-cloud-logging/README.md:305]:
```suggestion
| Cloud Logging | [OpenSearch cluster health](https://raw.githubusercontent.com/kyma-project/telemetry-manager/main/docs/user/integration/sap-cloud-logging/alert-health.json) | The OpenSearch cluster might become unhealthy, which is indicated by a "red" status using the [cluster health API](https://opensearch.org/docs/1.3/api-reference/cluster-api/cluster-health).|
```

File: docs/user/integration/sap-cloud-logging/README.md [hunk 6]
```diff
@@ -284,3 +285,31 @@ For easier access, add a navigation node to the Observability section as well as
...
+| Kyma Telemetry Integration | [Application log ingestion](https://raw.githubusercontent.com/kyma-project/telemetry-manager/main/docs/user/integration/sap-cloud-logging/alert-app-log-ingestion.json) | The LogPipeline for shipping [application logs](#ship-logs-to-sap-cloud-logging) might loose connectivity to SAP Cloud Logging resulting in no application logs being ingested anymore.|
```
Comment (NHingerl) [inline docs/user/integration/sap-cloud-logging/README.md:306]:
```suggestion
| Kyma Telemetry Integration | [Application log ingestion](https://raw.githubusercontent.com/kyma-project/telemetry-manager/main/docs/user/integration/sap-cloud-logging/alert-app-log-ingestion.json) | The LogPipeline for shipping [application logs](#ship-logs-to-sap-cloud-logging) might lose connectivity to SAP Cloud Logging, with the effect that no application logs are ingested anymore.|
```

File: docs/user/integration/sap-cloud-logging/README.md [hunk 7]
```diff
@@ -284,3 +285,31 @@ For easier access, add a navigation node to the Observability section as well as
...
+| Kyma Telemetry Integration | [Access log ingestion](https://raw.githubusercontent.com/kyma-project/telemetry-manager/main/docs/user/integration/sap-cloud-logging/alert-access-log-ingestion.json) | The LogPipeline for shipping [access logs](#ship-logs-to-sap-cloud-logging) might loose connectivity to SAP Cloud Logging resulting in no application logs being ingested anymore. |
```
Comment (NHingerl) [inline docs/user/integration/sap-cloud-logging/README.md:307]:
```suggestion
| Kyma Telemetry Integration | [Access log ingestion](https://raw.githubusercontent.com/kyma-project/telemetry-manager/main/docs/user/integration/sap-cloud-logging/alert-access-log-ingestion.json) | The LogPipeline for shipping [access logs](#ship-logs-to-sap-cloud-logging) might lose connectivity to SAP Cloud Logging, with the effect that no application logs are ingested anymore. |
```

File: docs/user/integration/sap-cloud-logging/README.md [hunk 8]
```diff
@@ -284,3 +285,31 @@ For easier access, add a navigation node to the Observability section as well as
...
+| Kyma Telemetry Integration | [Trace ingestion](https://raw.githubusercontent.com/kyma-project/telemetry-manager/main/docs/user/integration/sap-cloud-logging/alert-trace-ingestion.json) | The TracePipeline for shipping [traces](#ship-distributed-traces-to-sap-cloud-logging) might loose connectivity to SAP Cloud Logging resulting in no traces being ingested anymore. |
```
Comment (NHingerl) [inline docs/user/integration/sap-cloud-logging/README.md:308]:
```suggestion
| Kyma Telemetry Integration | [Trace ingestion](https://raw.githubusercontent.com/kyma-project/telemetry-manager/main/docs/user/integration/sap-cloud-logging/alert-trace-ingestion.json) | The TracePipeline for shipping [traces](#ship-distributed-traces-to-sap-cloud-logging) might lose connectivity to SAP Cloud Logging, with the effect that no traces are ingested anymore. |
```

File: docs/user/integration/sap-cloud-logging/README.md [hunk 9]
```diff
@@ -284,3 +285,31 @@ For easier access, add a navigation node to the Observability section as well as
...
+| Kyma Telemetry Integration | [Metric ingestion](https://raw.githubusercontent.com/kyma-project/telemetry-manager/main/docs/user/integration/sap-cloud-logging/alert-metric-ingestion.json) | The MetricPipeline for shipping [metrics](#ship-metrics-to-sap-cloud-logging) might loose connectivity to SAP Cloud Logging resulting in no metrics being ingested anymore. |
```
Comment (NHingerl) [inline docs/user/integration/sap-cloud-logging/README.md:309]:
```suggestion
| Kyma Telemetry Integration | [Metric ingestion](https://raw.githubusercontent.com/kyma-project/telemetry-manager/main/docs/user/integration/sap-cloud-logging/alert-metric-ingestion.json) | The MetricPipeline for shipping [metrics](#ship-metrics-to-sap-cloud-logging) might lose connectivity to SAP Cloud Logging, with the effect that no metrics are ingested anymore. |
```

File: docs/user/integration/sap-cloud-logging/README.md [hunk 10]
```diff
@@ -284,3 +285,31 @@ For easier access, add a navigation node to the Observability section as well as
...
+
+## SAP Cloud Logging Dashboards
+
+SAP Cloud Logging provides already an extensive set of Dashboards under `OpenSearch Dashboards > Dashboard` revealing insights in traffic and application logs. These dashboards are prefixed with `Kyma_` and are based on both kind of [log ingestion](#ship-logs-to-sap-cloud-logging) (application and access logs).
```
Comment (NHingerl) [inline docs/user/integration/sap-cloud-logging/README.md:313]:
```suggestion
SAP Cloud Logging provides an extensive set of dashboards under `OpenSearch Dashboards > Dashboard`, which give insights in traffic and application logs. These dashboards are prefixed with `Kyma_`, and are based on both kinds of [log ingestion](#ship-logs-to-sap-cloud-logging): application and access logs.
```

File: docs/user/integration/sap-cloud-logging/README.md [hunk 11]
```diff
@@ -284,3 +285,31 @@ For easier access, add a navigation node to the Observability section as well as
...
+
+Additionally you can introspect distributed traces under `OpenSearch Plugins > Observability`.
```
Comment (NHingerl) [inline docs/user/integration/sap-cloud-logging/README.md:315]:
```suggestion
Additionally, you can view distributed traces under `OpenSearch Plugins > Observability`.
```

File: docs/user/integration/sap-cloud-logging/README.md (hunk 3 above)
Comment (NHingerl) [reply docs/user/integration/sap-cloud-logging/README.md:299]: good idea!

**Review by chrkl — APPROVED** [review]

**Review by NHingerl — APPROVED** [review]

**Pull request merged**